kytos/cli/commands/bug_report/api.py

```diff
@@ -0,0 +1,99 @@
+"""Translate cli commands to non-cli code."""
+import re
+import subprocess
+
+
+class BugReportAPI:
+    """Class to retrieve Kytos environment informations.
+
+    Class is responsible to collect system information, python environment,
```

Comment:
`Collect system information, python environment, python packages and installed napps to print this report to the user aiming improve bug reports.`